test/commands-tests/history.js

```diff
@@ -303,6 +338,51 @@ describe('AEproject History', async () => {
             assert.isOk(hasFail && hasError, 'History log is not correct!');
         });
 
+        it('With account that has no aettos, deployment should be unsuccessful and should has an error', async () => {
+
+            let client = await cliUtils.getClient(network, moneyKeyPair);
+
+            // account should have minimum of 1 aettos 
+            // or will throw exception of "account not found"
+            await client.spend(1, keyPair.publicKey);
+
+            await linkPackages();
+
+            await execute(constants.cliCommands.DEPLOY, [
+                "--secretKey",
```

Comment:
might be good if we follow all other cli subcommands pattern, and the `--secretKey` be exported in constants